mkdocs.yml

```diff
@@ -5,14 +5,17 @@ docs_dir: 'markdown-docs'
 google_analytics: ['UA-991100-5', 'search.asf.alaska.edu']
 
 theme:
-    name: material
+    name: asf-theme
     custom_dir: asf-theme
     favicon: assets/images/favicon.ico
     logo: assets/images/ASFLogo-Blue2.png
 
 extra_css:
   - assets/stylesheets/extra.css
 
+extra_javascript:
+  - assets/javascripts/expert-rec-search.js
```

Comment:
You also don't need this (or any of the asf-theme files) as it's included in the theme:

https://github.com/ASFHyP3/mkdocs-asf-theme/blob/develop/asf_theme/assets/javascripts/expert-rec-search.js